<launch>
    <!-- Start rosbag play -->
    <executable
        cmd="ros2 bag play --clock 1000 $(find-pkg-share gyak3)/bag"
        output="screen"
        name="rosbag_play"
        shell="true"
    />

    <node pkg="gyak3" exec="path.py" name="path_node" output="screen">
        <param name="use_sim_time" value="true"/>
        <param name="path_topic_name" value="/path"/>
        <param name="odom_topic_name" value="/odom"/>
        <param name="max_size" value="500"/>
    </node>

    <node pkg="rviz2" exec="rviz2" name="rviz2" args="-d $(find-pkg-share gyak3)/rviz/gyak3.rviz"/>
</launch>